src/psyclone/psyir/transformations/omp_loop_trans.py

```diff
@@ -228,13 +228,15 @@ def _directive(self, children, collapse=None):
             node.reprod = self._reprod
         return node
 
-    def apply(self, node, options=None):
+    def apply(self, node, options=None,
+              reprod: bool = None,
+              **kwargs):
         '''Apply the OMPLoopTrans transformation to the specified PSyIR Loop.
 
         :param node: the supplied node to which we will apply the \
                      OMPLoopTrans transformation
         :type node: :py:class:`psyclone.psyir.nodes.Node`
-        :param options: a dictionary with options for transformations\
+        :param options: a dictionary with options for transformations
```

Comment:
We need a description of the `reprod` argument now.

Reply:
Added